src/utils/helpers.js

```diff
@@ -0,0 +1,12 @@
+export function setButtontext(
+  btn,
+  isLoading,
+  defaultText = "Save",
+  loadingText = "Saving..."
+) {
+  if (isLoading) {
+    btn.innerHTML = "Loading...";
+  } else {
+    btn.innerHTML = "Submit";
```

Comment:
`innerHTML` should not be used because it has some `vulnerability`. Please, use `textContent` for any text